sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/multi/MultiServiceEventClient.java

```diff
@@ -115,32 +250,21 @@ public void unregistered() {
      */
     @Override
     public OfferResponse offers(Collection<Protos.Offer> offers) {
-        // If uninstalling, then finish uninstall. Otherwise decline short.
-        boolean noClients = false;
-        // Decline short.
+        // Decline short if any client isn't ready.
         boolean anyClientsNotReady = false;
```

Comment:
I like the explicitness of anyServicesNotReady.

But they are technically MesosEventClients so /shrug.